README.md

```diff
@@ -1,32 +1,64 @@
 
 # Dialog Flow Framework
 
-The Dialog Flow Framework (DFF) allows you to write conversational services. The service is written by defining a special dialog graph that describes the behavior of the dialog service. The dialog graph contains the dialog script. DFF offers a specialized language (DSL) for quickly writing dialog graphs. You can use it in such services for writing skills for Amazon Alexa and etc, chat-bots for social networks, websites call-centers and etc. 
+The Dialog Flow Framework (DFF) allows you to write conversational services.
+The service is written by defining a special dialog graph that describes the behavior of the dialog service.
+The dialog graph contains the dialog script. DFF offers a specialized language (DSL) for quickly writing dialog graphs.
+You can use it in services such as writing skills for Amazon Alexa, etc., chatbots for social networks, website call centers, etc.
 
-[![Documentation Status](https://readthedocs.org/projects/dialog-flow-engine/badge/?version=latest)](https://readthedocs.org/projects/dialog-flow-engine/badge/?version=latest)
```

Comment:
Return documentation button.
```
[![Documentation Status](https://github.com/deeppavlov/dialog_flow_framework/workflows/build_and_publish_docs/badge.svg)](https://deeppavlov.github.io/dialog_flow_framework)
```
I don't know which link should be here:
[to the doc itself](https://deeppavlov.github.io/dialog_flow_framework)
or
[to the action page](https://github.com/deeppavlov/dialog_flow_framework/actions)

Reply:
I think the second one is better if we are talking about status.

Reply:
Where can i find button "docs" (without "failing")?

Reply:
The badge is set automatically -- if the workflow is failing the badge will display `failing` status. So to fix it the workflow has to be fixed.